packages/frontend/src/creation-form/i18n/en.json

```diff
@@ -1,8 +1,14 @@
 {
+    "info": "Need more info on how Reality.eth works? Check out the documentation page ",
+    "info.here": "here",
     "label.arbitrator": "Arbitrator:",
+    "info.arbitrator": "Anyone who wants to challenge the final answer but doesn't have enough funds to post it directly can request for an arbitrator to step in. Keep in mind the arbitrator can assign a cost for this service.",
     "label.reality.template": "Result type:",
+    "info.reality.template": "The  answer can be either yes or no or a specific number value depending on your needs.",
```

Comment:
Double space between `The  answer`.

Reply:
Addressed in 7900adc9180edc26915bab93ad3762c7b31abf7c